integration-tests/eks/validateResources_test.go

```diff
@@ -34,8 +34,9 @@ const (
 	fluentBitName        = "fluent-bit"
 	fluentBitNameWindows = "fluent-bit-windows"
 	dcgmExporterName     = "dcgm-exporter"
+	neuronMonitor        = "neuron-monitor"
 	podNameRegex         = "(" + agentName + "|" + agentNameWindows + "|" + operatorName + "|" + fluentBitName + "|" + fluentBitNameWindows + ")-*"
-	serviceNameRegex     = agentName + "(-headless|-monitoring)?|" + agentNameWindows + "(-headless|-monitoring)?|" + addOnName + "-webhook-service|" + dcgmExporterName + "-service"
+	serviceNameRegex     = agentName + "(-headless|-monitoring)?|" + agentNameWindows + "(-headless|-monitoring)?|" + addOnName + "-webhook-service|" + dcgmExporterName + "-service|" + neuronMonitor + "-service"
```

Comment:
non blocking nit: As we start adding more resources maybe we should consider just having an array with the exact names so that it's more explicit and easy to read